runtime/queries/comment/highlights.scm

```diff
@@ -0,0 +1,25 @@
+[
+ "("
+ ")"
+] @punctuation.bracket
+
+":" @punctuation.delimiter
+
+(tag (name) @ui.text (user)? @constant)
+
+((tag ((name) @warning))
+ (#any-of? @warning "TODO" "HACK" "WARNING"))
+
+("text" @warning
+ (#any-of? @warning "TODO" "HACK" "WARNING"))
+
+((tag ((name) @error))
+ (#any-of? @error "FIXME" "XXX" "BUG"))
+
+("text" @error
+ (#any-of? @error "FIXME" "XXX" "BUG"))
```

Comment:
`any-of?` isn't actually supported by us: https://github.com/helix-editor/helix/issues/887#event-5507096538

Should be possible to implement though

Reply:
you should be able to replace any `#any-of?` with an equivalent (but kinda ugly) `#match?` like here: https://github.com/helix-editor/helix/blob/6e62c3de47ff14df394c1c17fec103c134aa045c/runtime/queries/elixir/highlights.scm#L181-L184

it might be nice to add `#any-of?` eventually, I think I've seen it in a bunch of different languages' queries

Reply:
I don't think `#any-of?` is the issue. I tried removing everything in the comment highlight file but still it didn't get comment highlight as fallback color.